commander/src/bootstrapping/templates/lisk-template-ts/templates/app/src/commands/account/show.ts

```diff
@@ -0,0 +1 @@
+export { AccountShowCommand } from 'lisk-commander';
```

Comment:
For all the commands, it should be either consistently expose as `default` or as object.
ie) some file exposes as `default class` but some as object